docs/fastdds/statistics/dds_layer/domainParticipant.rst

```diff
@@ -38,11 +32,77 @@ This method requires as parameter:
 Obtain pointer to the extended |StatisticsDomainParticipant-api| class
 ----------------------------------------------------------------------
 
-.. warning::
-    This method will be implemented in future releases. For the moment it always returns ``nullptr``.
-
```

Comment:
Missing snippet

Reply:
I have added a complete code snippet using all the DDS API after explaining the API